metabase/metabase
# Fix - Text weirdly wrapped in column dropdown in Polish language

Closes #35047

### Description

Screenshots:
- real labels: [before](https://github.com/metabase/metabase/assets/6830683/3a1fe0b3-a1c8-41cd-aa68-6887b8a52bd7) & [after](https://github.com/metabase/metabase/assets/6830683/e9e6cdf8-f2f5-4619-bb17-ac59749361af)
- artificially long labels: [before](https://github.com/metabase/metabase/assets/6830683/1ee19e33-aef4-42fb-b440-43ffd4713934) & [after](https://github.com/metabase/metabase/assets/6830683/c60593d4-5f59-4883-ba2d-a9d3ec3871e9)
- English: [before](https://github.com/metabase/metabase/assets/6830683/d35ed160-af5a-4fcb-9a62-3800365fe0c6) & [after](https://github.com/metabase/metabase/assets/6830683/6d6303d3-7516-4285-8eb3-6027f5ca40ec) (identical, no change)

### How to verify

1. Switch app language to Polish
2. Create a new GUI question
3. Select Orders table from Sample Database as data source
4. Aggregate by rows count
5. Open breakout column dropdown

All dropdown menu items representing columns should have the same height (but they should wrap into multiple lines if they are in fact very long - this does not happen in the above testing scenario though).
Words in dropdown menu items should not be broken into multiple lines.
